### PR TITLE
[TR] interface_check.sh now only checks for use statements of entities

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: app/Resources/jenkins/interface_check.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ OUTPUT=''
 # Channel Completeness Currency Family Group Locale Metric Product ProductMedia ProductPrice ProductValue
 
 for pattern in Attribute Category Completeness Product ProductValue; do
-    OUTPUT=`find $1 -type f -name "*.php" | grep -v "/Tests/" | xargs grep -P "[^a-zA-Z']$pattern([ ',;)]|\$)" | grep -v "*" | grep -v "namespace" | grep -v "class $pattern" | tr "\n" "#"`$OUTPUT
+    OUTPUT=`find $1 -type f -name "*.php" | grep -v "/Tests/" | xargs grep -P "^use .*[^a-zA-Z']$pattern([ ',;)]|\$)" | grep -v "*" | grep -v "namespace" | grep -v "class $pattern" | tr "\n" "#"`$OUTPUT
 done
```
Comment (jmleroux) [inline app/Resources/jenkins/interface_check.sh]: @grena : if we check only `use` statements, then the filter can be simplified : 
```OUTPUT=`find $1 -type f -name "*.php" | grep -v "/Tests/" | xargs grep -P "^use .*[^a-zA-Z']$pattern([ ',;)]|\$)" | tr "\n" "#"`$OUTPUT```

Comment (grena) [inline app/Resources/jenkins/interface_check.sh]: In fact it finds a lot a false positive if I remove it ^^

**Comment by solivier** [comment]
GTM

**Comment by jmleroux** [comment]
GTM

**Pull request merged**